test/unit/path_spec.lua

```diff
@@ -293,6 +294,37 @@ describe('path_shorten_fname_if_possible', function()
   end)
 end)
 
+describe('path.c path_guess_exepath', function()
```

Comment:
Also need to check that if one `$PATH` item exceeds MAXPATHL tests do not crash.